docs/src/api.md

```diff
@@ -1,24 +1,5 @@
-# ChainRules API Documentation
+# API Documentation
 
-```@docs
-ChainRules.frule
-ChainRules.rrule
-ChainRules.AbstractRule
-ChainRules.Rule
-ChainRules.DNERule
-ChainRules.WirtingerRule
-ChainRules.accumulate
-ChainRules.accumulate!
-ChainRules.store!
-```
-
-```@docs
-ChainRules.AbstractDifferential
-ChainRules.extern
-ChainRules.Casted
-ChainRules.Wirtinger
-ChainRules.Thunk
-ChainRules.Zero
-ChainRules.DNE
-ChainRules.One
+```@autodocs
+Modules = [ChainRulesCore]
```

Comment:
The downside to this is that you lose the ability to order things explicitly, but :man_shrugging: